services/api/src/resources/organization/resolvers.ts

```diff
@@ -358,6 +358,33 @@ export const getProjectsByOrganizationId: ResolverFn = async (
   return rows;
 };
 
+export const getOrganizationByProject: ResolverFn = async (
+  project,
+  args,
+  { sqlClientPool, hasPermission },
+  info
+) => {
+
+  if (!project || !project.organization) {
+    return null
+  }
+
+  const extraFields = requestsExtraFields(info, ['id', 'name', 'friendlyName', 'description']);
+
+  let orgQuery = Sql.selectOrganizationDetails(project.organization);
+
+  if (extraFields) {
+    await hasPermission('organization', 'view', {
+      organization: project.organization,
+    });
+    orgQuery = Sql.selectOrganization(project.organization);
```

Comment:
This shouldn't be required. If a user without permission tries to request the extra fields, they'll get a permission denied and no data will be returned from the previous select on line 374

Reply:
We might want to rethink description as well, so for now I think remove it so its only id and name.

We can re-evaluate description at a later period.